exercises/difference-of-squares/HINTS.md

```diff
@@ -1,3 +1,4 @@
 ## Hints
 For this exercise the following F# feature comes in handy:
```

Comment:
@ErikSchierboom now there are two items in the list it needs to be "features come" instead of "feature comes" :)